.github/workflows/linux-qe-template.yml

```diff
@@ -46,10 +52,13 @@ jobs:
           if [[ "${{inputs.qe-type}}" == "e2e" ]]; then
             status_context="${status_context}-${{inputs.preset}}"
           fi
-          status_context="${status_context}/windows-${{matrix.windows-version}}-${{matrix.windows-featurepack}}"
+          status_context="${status_context}/Linux-ARM64"
           echo "status_context=${status_context}" >> "$GITHUB_ENV"
 
-          echo "${PULL_SECRET}" > pull-secret
+          version_line=$(cat Makefile | grep "OPENSHIFT_VERSION ?=")
```

Comment:
No very common but there were certain releases specially on new minor openshift versions where openshift and microshift versions are not in sync...better fix it now picking both versions as you already have the condition to check the right bundle

Reply:
Add the microshift bundle version